feat(vpn-port-forwarding): make documentation up-to-date (#134)
qdm12/gluetun-wiki: setup/advanced/vpn-port-forwarding.md

### qBittorrent example

`VPN_PORT_FORWARDING_UP_COMMAND=/bin/sh -c 'wget -O- --retry-connrefused --post-data "json={\"listen_port\":$(echo {{PORTS}} | cut -d, -f1),\"random_port\":false,\"upnp\":false}" http://127.0.0.1:8080/api/v2/app/setPreferences 2>&1'`

For this to work, the qBittorrent web UI server must be enabled and listening on port `8080` and the Web UI "Bypass authentication for clients on localhost" must be ticked (json key `bypass_local_auth`) so Gluetun can reach qBittorrent without authentication.

### Transmission example

Transmission API requires CSRF tokens, so check out [the `transmission-port-update.sh` script](https://github.com/qdm12/gluetun/pull/2611/files).

## Allow a forwarded port through the firewall

For non-native integrations where you have a designated forwarded port from your VPN provider, you can allow it by adding it to the environment variable `FIREWALL_VPN_INPUT_PORTS`.

## Port redirection using iptables

> Gluetun supports setting up a redirection of incoming traffic from VPN opened port to a custom localhost port of your choosing, by using the `VPN_PORT_FORWARDING_LISTENING_PORT` environmental variable. Do not use this with torrent clients, or any other software that publicly announces its port, as that software wouldn't know what the publicly visible port is, and would be announcing the private port instead.

Reviewer: ```suggestion
Gluetun supports setting up a redirection of incoming traffic from the VPN opened port to a custom localhost port of your choosing, by using the `VPN_PORT_FORWARDING_LISTENING_PORT` environment variable. Do not use this with torrent clients, or any other software that publicly announces its port, as that software would not be aware of the publicly visible port and would be announcing the private port instead.
```
